### PR TITLE
Issue #SH-689: NISHTHA - changes to course progress & score report

### PR DESCRIPTION
**Pull request by utk14**

### CODE REVIEW

File: data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala [hunk 1]
```diff
@@ -173,20 +173,28 @@ object AssessmentMetricsJobV2 extends optional.Application with IJob with BaseRe
         col("username"),
         col(UserCache.userchannel))
 
+    val userDenormDF = if (!reportId.toLowerCase.equalsIgnoreCase("nishtha-reports")) {
```
Comment (maheshkumargangula) [inline data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala:176]: Don't put this condition based on report Id. it should be a new configuration property. May be `all_channel_data` true. The default value will be false.

Comment (utk14) [reply data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala]: Done

File: data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala [hunk 2]
```diff
@@ -46,6 +46,9 @@ object AssessmentMetricsJobV2 extends optional.Application with IJob with BaseRe
   }
 
   private def execute(config: JobConfig, batchList: List[String])(implicit sc: SparkContext, fc: FrameworkContext) = {
+    val modelParams = config.modelParams.getOrElse(Map[String, AnyRef]())
+    val allChannelData: Boolean = modelParams.getOrElse("allChannelData", false).asInstanceOf[Boolean]
```
Comment (maheshkumargangula) [inline data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala:50]:
```suggestion
    val applyPrivacyPolicy: Boolean = modelParams.getOrElse("applyPrivacyPolicy", true).asInstanceOf[Boolean]
```

**Comment by utk14** [comment]
The task has been taken up by Isha

**Pull request closed without merging**